api/destination_credential.go

```diff
@@ -0,0 +1,67 @@
+package api
+
+import (
+	"github.com/infrahq/infra/uid"
+)
+
+type CreateDestinationCredential struct {
+	Destination string `form:"destination"`
+}
+
+type ListDestinationCredential struct {
+	Destination     string `form:"destination"`
+	LastUpdateIndex int64  `form:"lastUpdateIndex"`
+}
+
+type ListDestinationCredentialResponse struct {
+	Items          []DestinationCredential `json:"items"`
+	MaxUpdateIndex int64                   `json:"maxUpdateIndex"`
+}
+
+// DestinationCredential is the database DestinationCredential object, not a request/response struct
```

Comment:
Shouldn't this be in the data package if its not a request response struct?

Reply:
This isn't actually a database struct. fixed the comment. `DestinationCredential represents the request for credentials to a particular destination`